ui-manchette/src/components/OperationalPointList.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect } from 'react';
+import { OperationalPointType } from '../types/pathPropertiesTypes';
+import OperationalPoint from './OperationalPoint';
+import cx from 'classnames';
+import { positionKm } from './helpers';
+
+interface OperationalPointListProps {
+  operationalPoints: OperationalPointType[];
+  isProportionnal?: boolean;
+  zoom?: number;
+  width?: number;
+}
+
+const OperationalPointList: React.FC<OperationalPointListProps> = ({
+  operationalPoints,
+  isProportionnal = false,
+  zoom = 1,
+}) => {
+  const [operationalPointsToDisplay, setOperationalPointsToDisplay] = React.useState<
+    OperationalPointType[]
+  >([]);
+  const baseHeight = 32;
+
+  const operationalPointStyle = (op: OperationalPointType, nextOp: OperationalPointType) => {
+    if (isProportionnal) {
+      if (!nextOp) {
+        return { height: `${baseHeight}px` };
+      } else {
+        return { height: `${positionKm(nextOp.position - op.position) * zoom * 8}px` };
+      }
+    } else {
+      return { height: `${baseHeight * zoom}px` };
+    }
+  };
+
+  useEffect(() => {
+    const calcOperationalPointsToDisplay = () => {
+      if (isProportionnal) {
+        const result = operationalPoints.reduce(
+          (acc, nextOp) => {
+            const op = acc[acc.length - 1];
+            const diff = positionKm(nextOp.position - op.position);
+            if (diff * 8 * zoom >= baseHeight) {
+              acc.push(nextOp);
+            }
+            return acc;
+          },
+          [operationalPoints[0]]
+        );
+
+        const lastElement = operationalPoints[operationalPoints.length - 1];
+        if (result[result.length - 1] !== lastElement) {
```

Comment:
I think it would be better to use isEqual from lodash
```suggestion
        if (isEqual(result[result.length - 1], lastElement)) {
```